ascent/src/aggregators.rs

```diff
@@ -38,6 +38,31 @@ where N: Ord + Add + Clone + Sum<N>
 }
 
 /// returns the number of tuples
+/// 
+/// # Examples
+/// 
+/// ```
+/// # use std::collections::HashSet;
+/// # use ascent_macro::ascent_run;
+/// # use ascent::aggregators::count;
+/// let res = ascent_run!{
+///    relation edge(u32, u32);
+///    relation path(u32, u32);
+///    relation num_paths(usize);
+///    path(a, b) <-- edge(a, b);
+///    path(a, c) <-- path(a, b), edge(b, c);
+/// 
+///    edge(1, 2);
+///    edge(2, 3);
+///    edge(3, 4);
+/// 
+///    num_paths(n) <-- agg n = count() in path(_, _);
+/// };
+/// // This example program is expected to produce 6 paths.
+/// let actual = res.num_paths.into_iter().collect::<HashSet<_>>(); 
+/// let expected = [(6,)].into_iter().collect::<HashSet<_>>();
+/// assert_eq!(actual, expected);

```

Comment:
I think you can simplify it to something like `assert_eq!(res.num_paths[0].0, 6)` if you find it more readable.
`num_paths` is expected to have a single fact, so this should be OK.